lib/src/widgets/rive_animation.dart

```diff
@@ -7,6 +7,7 @@ enum _Source {
   asset,
   network,
   file,
+  riveFile,
```

Comment:
I guess this only internal so probably the naming is fine (if a bit confusing) wonder if "direct" or something makes more sense.